server/controllers/index.js

```diff
@@ -1,10 +1,14 @@
+// import https from 'https'
+
 import Katathon from '../models/katathonModel'
 
 import {
   dateToTimestamp,
   getNextEvent
 } from '../helpers'
 
+const https = require('https')
+
```

Comment:
Any preferences to require over import?
To be more consistent we can use import everywhere.?

Reply:
Agreed. Prefer ES2015 import

Reply:
Ah yes. This should be import.